src/main/java/org/alliancegenome/curation_api/services/helpers/annotations/AnnotationUniqueIdHelper.java

```diff
@@ -129,6 +125,11 @@
 		if (annotation.getDiseaseGeneticModifierRelation() != null) {
 			uniqueId.add(annotation.getDiseaseGeneticModifierRelation().getName());
 		}
+		if(annotation instanceof GeneDiseaseAnnotation gda){
+			if(gda.getSgdStrainBackground() != null){
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' is not followed by whitespace.